05-setting-up.Rmd

```diff
@@ -57,10 +185,32 @@ https://bookdown.org/ndphillips/YaRrr/projects-in-rstudio.html
 
 http://projecttemplate.net/
 
-### R Markdown files
+### Scientific notebooks (Rmd or qmd)
+
+The generous use and keeping of notebooks is a useful tool for documentation of the development of an analysis.
+
+Data analyses can lead one on a winding trail of decisions and side investigations, but notebooks allow you to narrate your thought process as you travel along these analyses explorations!
+
+```{r, fig.align='center', echo = FALSE, fig.alt= "Ruby is looking at her computer that has a lovely notebook with a heatmap! Ruby says ‘Working from this notebook allows me to interactively develop on my data analysis and write down my thoughts about the process all in one place!’"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1LMurysUhCjZb7DVF6KS9QmJ5NBjwWVjRn40MS9f2noE/edit#slide=id.gf8f405fdab_0_186")
+```
+
+**Your scientific notebook should include descriptions that describe:**   
+
+#### The purposes of the notebook
+
+What scientific question are you trying to answer? Describe the dataset you are using to try to answer this and why does it help answer this question?
+
+#### The rationales behind your decisions
+
+Describe why a particular code chunk is doing a particular thing -- the more odd the code looks, the greater need for you to describe why you are doing it.
+
+Describe any particular filters or cutoffs you are using and how did you decide on those?
+
+For data wrangling steps, why are you wrangling the data in such a way -- is this because a certain package you are using requires it?
```

Comment:
```suggestion
For data wrangling steps, describe why you are wrangling the data in such a way. Is this because a certain package you are using requires it?
```